src/API/Google/Query/MerchantPriceBenchmarksQuery.php

```diff
@@ -0,0 +1,52 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\API\Google\Query;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Google\Query\MerchantQuery;
+
+/**
+ * Class MerchantPriceBenchmarksQuery
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\API\Google\Query
+ */
+class MerchantPriceBenchmarksQuery extends MerchantQuery {
+
+	/**
+	 * MerchantPriceBenchmarksQuery constructor.
+	 */
+	public function __construct() {
```

Comment:
The constructor will need to handle `$args` so we can pass `next_page` and `per_page` args, like https://github.com/woocommerce/google-listings-and-ads/blob/925c3fb030447cdff4c10c0901c91a2ac300a282/src/API/Google/MerchantReport.php#L76-L81